cloudvolume/datasource/precomputed/image/__init__.py

```diff
@@ -392,25 +392,39 @@ def make_shard(self, img, bbox, mip=None, spec=None):
       else:
         raise ValueError("mip {} does not have a sharding specification.".format(mip))
 
+    bbox = Bbox.create(bbox)
     if bbox.subvoxel():
       raise ValueError("Bounding box is too small to make a shard. Got: {}".format(bbox))
 
-    bbox = Bbox.create(bbox, self.meta.bounds(mip), bounded=True)
+    # Alignment Checks:
+    # 1. Aligned to atomic chunks - required for grid point generation
     aligned_bbox = bbox.expand_to_chunk_size(
       self.meta.chunk_size(mip), offset=self.meta.voxel_offset(mip)
     )
-    aligned_bbox = Bbox.clamp(aligned_bbox, self.meta.bounds(mip))
-
     if bbox != aligned_bbox:
       raise exceptions.AlignmentError(
         "Unable to create shard from a non-chunk aligned bounding box. Requested: {}, Aligned: {}".format(
         bbox, aligned_bbox
       ))
 
+    # 2. Covers the dataset at least partially
+    aligned_bbox = Bbox.clamp(aligned_bbox, self.meta.bounds(mip))
+    if any(aligned_bbox.size3()) is False:
```

Comment:
I'd suggest `aligned_bbox.subvoxel()` to cover cases where it's less than one voxel but more than zero.